### PR TITLE
ENG-30 provide feedback if DEs or corresponding components are not presented

### PR DESCRIPTION
**Pull request by ichalagashvili**
- Text visual feedbacks are provided

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/state/digital-exchange/components/actions.js [hunk 1]
```diff
@@ -268,11 +270,13 @@ export const fetchDEComponents = (paginationMetadata = {
           dispatch(setPage(data.metaData));
         } else {
           dispatch(addErrors(data.errors.map(err => err.message)));
+          data.errors.forEach(err => dispatch(addToast(err.message, TOAST_ERROR)));
```
Comment (gidesan) [inline src/state/digital-exchange/components/actions.js]: I remembered we preferred to have one toast for multiple errors. @nicpuddu can you please confirm that?

Comment (nicpuddu) [reply src/state/digital-exchange/components/actions.js]: this only makes sense (multiple toasts) if the errors are not related to one another... if they are all tied together it should be a single toast.

Comment (nicpuddu) [reply src/state/digital-exchange/components/actions.js]: on top of that the errors should be human readable and not technical mumbo jumbo, remember that.

Comment (ichalagashvili) [reply src/state/digital-exchange/components/actions.js]: DE components error handling is another story which is blocked by BE and will be implemented in the future, so I think we can accept this atm.

Comment (gidesan) [reply src/state/digital-exchange/components/actions.js]: @ichalagashvili is there any update on that?

Comment (ichalagashvili) [reply src/state/digital-exchange/components/actions.js]: So, I investigated it and errors messages here are independent. So, multiple toasts make sense. (DE component A can be failed for reason X and DE component B can be failed for another reason Y). But, in general, maybe the FETCHING logic for DE components can be changed, or at least the error reporting logic. See: https://jira.entando.org/browse/ENG-93

Comment (gidesan) [reply src/state/digital-exchange/components/actions.js]: there are lots of DE changes planned, we can accept the PR as it is atm

**Pull request merged**